.travis.yml

```diff
@@ -5,4 +5,10 @@ cache: bundler
 before_install: gem install bundler
 script: bundle exec rspec
 rvm:
+  - ruby-head
+  - 2.4.2
   - 2.4.0
+  - 2.3.5
+  - 2.3.1
```

Comment:
Is there any reason to support 2.3.1 and 2.2.0 if 2.3.5 and 2.2.5 supported?

Minor ruby version should always be updated after new release anyway

Reply:
It is just test PR, and you can don't pay attension on it